src/Properties/PublishProfiles/FolderProfile.pubxml

```diff
@@ -0,0 +1,13 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
this file should not be checked in

Reply:
We've created a NuGet package with those changes and I have let it there by mistake. 
fixed, by removing the folder.